package.json

```
@@ -8,6 +8,7 @@
     "@mui/material": "5.17.1",
     "@mui/utils": "5.17.1",
     "@mui/x-date-pickers": "^7.29.4",
+    "@mui/system": "^5.17.1",
```

Minor nit: we usually try to pin dependencies exactly. Looks like we missed the version on line 10.
```suggestion
    "@mui/system": "5.17.1",
```